x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/unpack_processor_events.ts

```diff
@@ -36,6 +36,7 @@ export function unpackProcessorEvents(
     body: { query: { bool: { filter: ESFilter[] } } };
   } = defaultsDeep(cloneDeep(params), {
     body: {
+      size: 0,
```

Comment:
I don't think we should set this here. IMHO it's dangerous to set a default of 0 here (the ES default is 10). Is the issue that TS will complain about a missing parameter?

Reply:
@dgieselaar yes, TS complains about the missing parameter, should I set it to `size:10` instead 0?

Reply:
no, I don't think we should set it here. We should fix the types that are expected here.

Reply:
The type that was failing was the one in the test when I got rid of the size there, but that is expected because the request type is `APMEventESSearchRequest | APMEventESTermsEnumRequest`, so we can delete the size in `unpack_processor_events.ts` 👍